index.js

```diff
@@ -14,11 +14,7 @@ program.usage('[options]').parse(process.argv)
 
 const app = express()
 
-app.use(bodyParser.text({
-    inflate: true,
-    limit: '2048kb',
-    type: '*/*',
-}))
+app.use(bodyParser.json());
```

Comment:
can we do both? would like to avoid a breaking change here (i.e. still use text/plain, add application/json, and let content-type decide)

```js
const maxReqBodySize = '2048kb' // or even better, get this from env
app
  .use(
    bodyParser.text(limit: maxReqBodySize),
    bodyParser.json(limit: maxReqBodySize)
  )
```